components/skeleton/skeleton-mixin.js

```diff
@@ -156,8 +156,7 @@ export const SkeletonMixin = dedupeMixin(superclass => class extends RtlMixin(su
 			 * Render the component as a [skeleton loader](https://github.com/BrightspaceUI/core/tree/main/components/skeleton).
 			 * @type {boolean}
 			 */
-			skeleton: { reflect: true, type: Boolean  },
-			_skeletonActive: { state: true },
```

Comment:
Since we're manually triggering an update whenever we know `skeleton`'s state has changed, we don't need modifying `_skeletonActive` to also trigger an update.